frontend/src/components/ArticleSection/index.tsx

```diff
@@ -37,9 +37,15 @@ export default function Article(props: ArticleProps) {
   };
 
   const generateKeywords = () => {
-    const temp = props.body[0].split(' ');
-    setkeywords(['Examples:'].concat(temp.slice(0, 3)));
-    //TODO: connect with BE
+    const requestOptions = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ text: props.body.join(), numOfKeywords: 3})
```

Comment:
Is 3 an arbitrary number? Or have we decided on 3 being the best number of keywords to use?

Reply:
3 is an argument that we can change in the future. Agree with Armin that more keywords might affect the visual appeal.